pom.xml

```diff
@@ -114,7 +114,7 @@ THE SOFTWARE.
     <!-- TODO: JENKINS-36716 - Switch to Medium once FindBugs is cleaned up, 430 issues on Mar 10, 2018 -->
     <findbugs.effort>Max</findbugs.effort>
     <findbugs.threshold>High</findbugs.threshold>
-    <findbugs.excludeFilterFile>../src/findbugs/findbugs-excludes.xml</findbugs.excludeFilterFile>
+    <findbugs.excludeFilterFile>${project.basedir}/../src/findbugs/findbugs-excludes.xml</findbugs.excludeFilterFile>
```

Comment:
Without this, `mvn -am -pl cli clean install -e` fails [here](https://github.com/gleclaire/findbugs-maven-plugin/blob/findbugs-maven-plugin-3.0.5/src/main/groovy/org/codehaus/mojo/findbugs/FindBugsMojo.groovy#L1331) with

```
org.codehaus.plexus.resource.loader.ResourceNotFoundException: Could not find resource '../src/findbugs/findbugs-excludes.xml'.
    at org.codehaus.plexus.resource.DefaultResourceManager.getResource (DefaultResourceManager.java:171)
    at org.codehaus.plexus.resource.DefaultResourceManager.getResourceAsInputStream (DefaultResourceManager.java:67)
    at org.codehaus.plexus.resource.ResourceManager$getResourceAsInputStream$1.call (Unknown Source)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo.getResourceAsFile (FindBugsMojo.groovy:1331)
    at org.codehaus.mojo.findbugs.FindBugsMojo$getResourceAsFile.callCurrent (Unknown Source)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo.getResourceFile (FindBugsMojo.groovy:1225)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo$_getFindbugsArgs_closure5.doCall (FindBugsMojo.groovy:889)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo.getFindbugsArgs (FindBugsMojo.groovy:887)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo.executeFindbugs (FindBugsMojo.groovy:1019)
    at …
    at org.codehaus.mojo.findbugs.FindBugsMojo.canGenerateReport (FindBugsMojo.groovy:549)
    at org.codehaus.mojo.findbugs.FindBugsMojo.execute (FindBugsMojo.groovy:688)
```

I am not exactly sure why, but anyway this parameter is documented to be “a file”, with no mention of whether relative paths are supported and, if so, what they would mean, so explicitly resolving against `project.basedir` seems wiser.